src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/CreatedBy.java

```diff
@@ -35,7 +35,15 @@ public String getFullName() {
         return fullName;
     }
 
+    public void setFullName(String fullName) {
```

Comment:
Do we need setters for these? id and email don't have them, if possible we should leave created by as an immutable object

Reply:
Yes we'll want them - the patcher needs them. Not sure the price is right for keeping it immutable we'd have to create a new object for each field in te patcher.